components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/SalesforceEndpointConfig.java

```diff
@@ -166,8 +167,10 @@
     private String instanceId;
 
     // Streaming API properties
-    @UriParam
-    private Long defaultReplayId;
+    @UriParam(description = "Default replayId setting if no value is found in " +
+                            "{@link #initialReplayIdMap}",
```

Comment:
Can you remove this @link as this is not javadoc and we want to keep those descriptions just text based.

Reply:
Ah, nice catch.